documentation/meta/contribution/good_first_and_help_wanted_issues.md

```diff
@@ -0,0 +1,194 @@
+
+Descriptions should:
+
+- Clearly articulate the expected outcome of resolving the issue. Include any of
+  the following, when relevant:
+  - Changes (modifications or additions) to tests
+  - Link to instructions for updating Playwright snapshots if the change might
+    require it
+  - Changes (modifications or additions) to documentation
+- Link to the relevant files and/or lines of code which need changing
+- Reference any past PRs which will help the contributor, for example:
+  - A PR with a similar change
+  - A PR which originally implemented the feature being modified
+- Relevant Openverse domain knowledge, either links to succinct documentation or
+  links and a summary of the important aspects when documentation is detailed
+  - Ideally "good first issue"s require essentially no Openverse-specific domain
+    knowledge, but if they at all do (for example, to explain why a non-obvious
+    solution is requested) then it must be included
+
+### Additional boilerplate for good first issue templates
+
+In addition to our standard issue template, good first issues should _also_
+contain the following block of requirements:
+
+```md
+## Good first issue checks
+
+- Have I filled out the PR template correctly?
+  - Did I include testing instructions?
+- Does my PR pass linting? (Test either via precommit or by running `just lint`
+  manually)
+- Do all the tests still pass?
+  - If JavaScript changes, run `pnpm -r run test`
+  - If API changes, run `just api/test`
+  - If catalog changes, run `just catalog/test`
+```
+
+## On "help wanted" issues
+
+All good first issues are naturally "help wanted" issues as well. "Help wanted"
+issues which do not also include the "good first issue" label are excellent
+candidates for second, third, fourth, and so on issues from repeat contributors.
+If a contributor is labeled as a "first time contributor" in the GitHub user
+interface, make sure the issue they are working on is indeed marked "good first
+issue", or that it is otherwise clear that they have advanced knowledge of the
+issue's problem space that makes them an appropriate candidate.
+
+For example, the maintainer of a 3rd party library we use might see we are
+having an issue with their code and offer a PR with a fix. This would be
+appropriate given their expertise.
+
+## Timing and process
+
+It can often be tricky to determine how and when to make requests of a community
+contributor. The following table provides guidelines for specific scenarios.
+
+Response times may be from the contributor _or_ from a maintainer depending on
+the situation.
+
+| Scenario                                     | Recommended Response Time (Days)                                 |
+| -------------------------------------------- | ---------------------------------------------------------------- |
+| New PR submitted                             | Typically priority based, but for community PRs should be 3 days |
```

Comment:
Should we update the PR review reminder bot to check the PR author, and use a 3 day expectation for anyone outside the maintainers list?

Reply:
This is a great idea! Probably best to do it in this PR rather than follow-up, so I'll try and add that asap.

Reply:
Actually, given my AFK, I'll make a separate issue for this. It requires adding a new method to our GitHubApi class in the catalog and making small but non-trivial changes to the pr review reminder dag that I unfortunately don't have time for.